.github/workflows/release.yml

```diff
@@ -41,26 +78,40 @@ jobs:
             saucelabs/stt-testcafe-node:${{ steps.prep.outputs.version }}
           build-args: |
             BUILD_TAG=${{ steps.prep.outputs.version }}
+
   release-windows-bundle:
     runs-on: windows-latest
+    needs: [create-release-draft]
     steps:
+      - name: Find draft tag
```

Comment:
Can this get messed up if two people create their own draft releases around the same time for test purposes?

Reply:
> Can this get messed up if two people create their own draft releases around the same time for test purposes?
@alexplischke  Yes, that could be one of the caveat.

I think that using in conjunction `GITHUB_RUN_ID` and `GITHUB_RUN_NUMBER` may mitigate.
It should protecting against this issue.

> https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

I also need to update to support (noticed that missing part today):
- Releasing a pre-release
- PreRelease from a non-master branch
- Disabling commit in case of pre-release

Reply:
Lately we had a lot of features added that fixed one problem but caused more issues than they initially fixed.
Releases/PreReleases need to work from any branch. This is important if we needed to go back and patch things up.
That also means that multiple releases around the same time from different branches should be supported as well ℹ️

Reply:
Added support for pre-release, and for multiple branch support.
Also now it check only for the draft of the current pipeline, and not the first one found.

More demos here:
https://github.com/FriggaHel/sauce-testcafe-runner/releases